src/main/scala/it/pagopa/interop/apigateway/api/impl/GatewayApiServiceImpl.scala

```diff
@@ -300,23 +247,22 @@ final case class GatewayApiServiceImpl(
     toEntityMarshallerEServiceDescriptors: ToEntityMarshaller[EServiceDescriptors],
     toEntityMarshallerProblem: ToEntityMarshaller[Problem]
   ): Route = authorize {
+    val operationLabel = s"Retrieving Descriptors of EService $eServiceId"
+    logger.info(operationLabel)
+
     val result: Future[EServiceDescriptors] = for {
       eServiceUUID <- eServiceId.toFutureUUID
       eService     <- catalogManagementService.getEService(eServiceUUID)(contexts)
-      descriptors  <- eService.descriptors.traverse(_.toModel).toFuture
+      descriptors  <- eService.descriptors
+        .filter(_.state != CatalogManagementDescriptorState.DRAFT)
```

Comment:
Bug

Same [here](https://github.com/pagopa/interop-be-api-gateway/pull/133/files?diff=split&w=1#r1049443007)